### PR TITLE
Updating Userguide with allocated features up to v2.0

### PR DESCRIPTION
**Pull request by CindyTsai1**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -56,24 +56,41 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 Format: `help`
 
 === Adding a person: `add`
+==== Adding a person manually
 
 Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` +
```
Comment (zacharytang) [inline docs/UserGuide.adoc:61]: The square brackets shouldn't be removed, as they denote an optional argument in the command

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -56,24 +56,41 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/TAG b/BIRTHDAY...` +
 Alternatives: `a` , `insert`
 
-[TIP]
-A person can have any number of tags (including 0)
+****
```
Comment (zacharytang) [inline docs/UserGuide.adoc]: Maybe separate the formatting for birthday into another point, leaving the tip intact?

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -56,24 +56,41 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
 Examples:
 
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
+* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 b/19980523`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 
+==== Adding a person using the information on social media _(Coming in v2.0)_
+
+Adds a person to address book from the social media accounts +
+Format: `add s/SOCIAL_MEDIA_TYPE SOCIAL_MEDIA_ID`
```
Comment (zacharytang) [inline docs/UserGuide.adoc]: Might be good to list the supported social media types

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -56,24 +56,41 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
+
+Examples:
+
+* `add s/facebook John Doe`
+
 === Listing persons : `list`
 
 Shows a list of all persons in the address book. +
 If a tag, or tags, is/are specified, then it shows a list of all persons containing one of the tags. +
-Format: `list [t/TAG]...` +
```
Comment (zacharytang) [inline docs/UserGuide.adoc:76]: Same as before, take note of the usage of the square bracket notation

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -82,14 +99,14 @@ Examples :
...
-
-
+* `list b/December` +
+Lists all persons in the address book having birthdays in `December`
 
 === Editing a person or a tag: `edit`
 ==== Editing a person
 
 Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...` +
```
Comment (zacharytang) [inline docs/UserGuide.adoc]: Here too

File: docs/UserGuide.adoc [hunk 6]
```diff
@@ -375,8 +427,12 @@ There is no need to save manually.
 == Other features
 Here are some features in the Address Book that are not command line based, but good to know to aid in your productivity
 
+=== Command Suggestion
+Typeing the wrong word for a command will trigger the address book to suggest the correct phrasing for that command
```
Comment (zacharytang) [inline docs/UserGuide.adoc]: Typo here, should be typing instead of typeing. Also, may be good to have an example?

**Review by zacharytang — CHANGES_REQUESTED** [review]
Removal of square brackets changes the meaning of some of the formats, may want to look into that. Good job overall

File: docs/UserGuide.adoc (hunk 1 above)
Comment (CindyTsai1) [reply docs/UserGuide.adoc:61]: ok!

**Review by zacharytang — APPROVED** [review]

**Pull request merged**